composer.json

```diff
@@ -4,29 +4,23 @@
     "type": "drupal-module",
     "license": "GPL-2.0-or-later",
     "require": {
-        "drupal/jsonapi": "^1.24",
-        "drupal/jsonapi_extras": "^2.5",
+        "drupal/jsonapi": "^2.0",
+        "drupal/jsonapi_extras": "^3.3",
         "drupal/schemata": "^1.0-alpha2",
-        "drupal/openapi": "^1.0-alpha1",
-        "dpc-sdp/tide_core": "^1.0"
+        "dpc-sdp/tide_core": "dev-feature/SDPA-1213_metatag_upgrade as 1.0",
+        "dpc-sdp/tide_site": "dev-feature/SDPA-1213_jsonapi_upgrade as 1.0"
```

Comment:
@enzolutions @kurtfoster tide_site is not a dependency of tide_api.

Reply:
@sonnykt , imho they shouldn't depend on each other. Unfortunately they have been coupled together now. I guess for now they need to depend on each other and there needs to be a ticket to decouple them.